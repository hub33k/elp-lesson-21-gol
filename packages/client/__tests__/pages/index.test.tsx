import { render, screen } from '@testing-library/react';
import HomePage from '~/pages/index';

describe(HomePage.name, () => {
  it('renders a heading', () => {
    render(<HomePage />);

    const heading = screen.getByRole('heading', {
      name: /Home page/i,
      level: 1,
    });

    expect(heading).toBeInTheDocument();
  });

  it('match snapshot', () => {
    const { container } = render(<HomePage />);
    expect(container).toMatchSnapshot();
  });
});
