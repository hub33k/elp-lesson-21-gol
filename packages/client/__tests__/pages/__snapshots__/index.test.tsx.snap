// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage match snapshot 1`] = `
<div>
  <h1>
    Home page
  </h1>
</div>
`;
